client/src/App.tsx

```diff
@@ -1,11 +1,15 @@
 import Console from "./components/Console";
+import Document from "./components/Document";
+import { NotFound } from "./components/NotFound";
 import { Route, Routes } from 'react-router-dom';
 
 export default function App() {
 
   return (
     <Routes>
       <Route path="/" element={<Console />} />
+      <Route path="/documents/:id" element={<Document />} />
+      <Route path="/page-not-found" element={<NotFound />} />
```

Comment:
probably it's better to use path="*"  also style it a bit

Reply:
you're right